src/shared/components/Header/__tests__/Header.test.tsx

```diff
@@ -1,17 +1,26 @@
 import * as React from 'react';
 import { shallow, mount } from 'enzyme';
-import Header from '..';
+import Header, {  serviceVersions } from '..';
+
+const versions:serviceVersions = {
```

Comment:
there are not spaces between your variable and your type. Are you running prettier?

Reply:
I have updated the code.